manifests/server/options.pp

```diff
@@ -70,6 +84,14 @@
   }
   validate_array($allow_query)
 
+  if !is_numeric($statistic_channel_port) {
+    fail('The statistic_channel_port is not a number')
+  }
+
+  if !validate_ip($statistic_channel_ip) {
```

Comment:
I think validate_ip auto fails, like validate_array or whatever, The extra if is extra